packages/web/src/pages/track-page/components/desktop/RemixContestSubmissionsTab.tsx

```diff
@@ -120,7 +120,7 @@ const RemixContestSubmissions = ({
   const remixesRoute = trackRemixesPage(permalink ?? '')
 
   return (
-    <Flex w='100%' column gap='2xl' p='xl'>
```

Comment:
lost column here, was this why the button was appearing next to the submissions?

Reply:
brilliant